src/entrypoint/cron.ts

```diff
@@ -1,15 +1,44 @@
 import { setupMongoConnection } from "../mongodb";
 import { baseLogger } from "../logger";
-import { updateEscrows, updateUsersPendingPayment } from "../ledger/balanceSheet"
+import { updateUsersPendingPayment } from "../ledger/balanceSheet"
 import { SpecterWallet } from "../SpecterWallet";
-import { updateRoutingFees } from "../lndUtils";
+import { offchainLnds, updateEscrows, updateRoutingFees } from "../lndUtils";
+import { InvoiceUser } from "../schema";
+
+// FIXME use lightning instead
+import { deleteFailedPayments } from "ln-service"
+
+const deleteExpiredInvoices = async () => {
+  // this should be longer than the invoice validity time
+  const delta = 2 // days
+  
+  const date = new Date();
+  date.setDate(date.getDate() - delta);
+  InvoiceUser.deleteMany({timestamp: {lt: date}})
+}
+
+const deleteFailedPaymentsAllLnds = async () => {
```

Comment:
This and `deleteExpiredInvoices` should be moved to lndUtils imo

Reply:
yup